src/lib/inputs.js

```diff
@@ -1,22 +1,33 @@
 import process from 'process'
 
 import core from '@actions/core'
+import { context } from '@actions/github'
+
+const getPrNumber = () => {
+  const { eventName, payload } = context
+
+  if (eventName === 'pull_request') {
+    return payload.number
+  }
+
+  if (eventName === 'workflow_run' && payload?.workflow_run?.pull_requests?.length === 1) {
```

Comment:
https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#workflow_run